.github/workflows/ci.yml

```diff
@@ -30,3 +30,20 @@ jobs:
           NEPTUNE_API_TOKEN: ${{ secrets.E2E_NEPTUNE_API_TOKEN }}
           NEPTUNE_PROJECT: e2e-tests/e2e
         uses: ./.github/actions/e2e
+  publish:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Build and test
```

Comment:
It says "build and test" but does not test. Can we make it conditional on passing tests?

Reply:
Good point. Will try in a moment

Reply:
How about now?